src/vs/workbench/contrib/chat/browser/chatWidget.ts

```diff
@@ -167,6 +168,27 @@ export class ChatWidget extends Disposable implements IChatWidget {
 		this.inputPart.focus();
 	}
 
+	focusWithId(id: string, type: 'next' | 'previous'): void {
```

Comment:
Why is this based on IDs when you started with an object anyway?

Reply:
Would prefer a name like `moveFocus` or separate `focusNext/focusPrevious`

Reply:
yeah so the focus changes a few times throughout the creation of the view. we need to use the id or it always thinks the last one is focused.